### ISSUE TITLE
Steps to implement code

### ISSUE THREAD

**Issue by NakitaOza** [Neooolee/WHUS2-CD#2]
Hi! I read your paper and the results are very promising! I am currently pursuing my master thesis in the same field of cloud detection. I see that you have uploaded some training code in this repository. I would like to reproduce these results. I did not find any evaluation/testing function. Is there some where wherein I can input images to the 'trained' algorithm and see the results of masking on the given input files?

**Comment by Neooolee** [comment]
The evaluation/testing process is the same as other semantic segmentation algorithms such as U-Net. Just need to replace the algorithm code (one line) with our code and loading the pre-trained model. Then run the evaluation/testing code as you do with U-Net.

**Comment by ludongxing** [comment]
hello! Is there any detailed description of the procedure execution order？?please

**Comment by Neooolee** [comment]
Just replace the file path in Autoprocess.py, then run Autoprocess.py.
